newIDE/app/src/ObjectGroupEditor/index.js

```diff
@@ -63,57 +64,61 @@ const ObjectGroupEditor = ({
       else type = '';
     });
 
-    let message = '';
-    if (type === undefined) {
-      message = 'This group is empty';
-    } else if (type === '') {
-      message =
-        "This group contains objects of different kinds. You'll only be able to use actions and conditions common to all objects with this group.";
-    } else {
-      message = `This group contains objects of the same kind (${type}). You can use actions and conditions related to this kind of objects in events with this group.`;
-    }
+    const message =
+      type === undefined ? (
+        <Trans>'This group is empty'</Trans>
+      ) : type === '' ? (
+        <Trans>
+          This group contains objects of different kinds. You'll only be able to
+          use actions and conditions common to all objects with this group.
+        </Trans>
+      ) : (
+        <Trans>
+          This group contains objects of the same kind ({type}). You can use
+          actions and conditions related to this kind of objects in events with
+          this group.
+        </Trans>
+      );
 
-    return <EmptyMessage>{message}</EmptyMessage>;
+    return <AlertMessage kind="info">{message}</AlertMessage>;
```

Comment:
If you change it to AlertMessage, I'm not sure that the `This group is empty` is useful, someone might think "Duh, I can see that"